### PR TITLE
Use django-storages-redux S3Boto for media

### PR DESCRIPTION
**Pull request by jgmize**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
This is a quick rebase.

File: masterfirefoxos/settings/base.py [hunk 1]
```diff
@@ -186,3 +186,8 @@
 
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 USE_X_FORWARDED_HOST = True
+
+DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
```
Comment (glogiotatidis) [inline masterfirefoxos/settings/base.py]: We can change this to something like:
```
if AWS_ACCESS_KEY_ID:
  DEFAULT_FILE_STORAGE = '...S3..'
```
for a friendlier dev env

**Comment by glogiotatidis** [comment]
Tested locally and it works!
Things to note:
 - Files don't get deleted from S3 when delete from /admin
 - [Storages documentation](http://django-storages.readthedocs.org/en/latest/index.html#installation) suggests that we add `storages` in `INSTALLED_APPS`

**Comment by glogiotatidis** [comment]
r+wc to make the change in settings. :dancer:

**Comment by jgmize** [comment]
Thanks @glogiotatidis, I've rebased and updated settings.

**Pull request merged**